iina/PlayerCore.swift

```diff
@@ -443,6 +443,16 @@ class PlayerCore: NSObject {
     sendOSD(.abLoop(info.abLoopStatus))
   }
 
+  func clearAbLoop() {
+    if let resetOption = mpv.getString(MPVOption.ProgramBehavior.resetOnNextFile) {
+        if resetOption.contains(MPVOption.PlaybackControl.abLoopA) &&
+                   resetOption.contains(MPVOption.PlaybackControl.abLoopB) {
```

Comment:
There's no problem with this approach, but I still prefer checking whether the two loop points are set (and set info.abLoopStatus to 0, 1, or 2 accordingly), because it also handles the case where A-B loop status is loaded from watch_later file by mpv.

Reply:
Sure.